.github/workflows/environment.yml

```
@@ -0,0 +1,21 @@
+name: Environment Suite
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  VERBOSE: true
+  ENVIRONMENT_UPSTREAM: docker.pkg.github.com/timberio/vector/environment:${{ github.sha }}
+
+jobs:
+  publish-new-environment:
+    runs-on: ubuntu-latest
+    env:
+      ENVIRONMENT_AUTOBUILD: true
+    steps:
+      - run: docker login https://docker.pkg.github.com -u hoverbear -p ${{ secrets.GITHUB_TOKEN }}
```

How about something like this?

```suggestion
      - run: docker login https://docker.pkg.github.com -u "$GITHUB_ACTOR" -p "$GITHUB_TOKEN"
```

Plus `docker.pkg.github.com/$GITHUB_REPOSITORY/environment` for image name.

---

Oh nice thank you!